docs/build/basics/scripts.md

```diff
@@ -87,6 +87,19 @@ flow scripts execute ./helloWorld.cdc
 
 A user can define their own scripts or can use already defined scripts by the contract authors that can be found by using the [FLIX](../../tools/flow-cli/flix) service.
 
+## Recommendations
```

Comment:
`Best Practices` i think could be a better title.

Reply:
yup - updated.